docs/get-involved/redis-live/index-redis-live.mdx

```diff
@@ -53,7 +53,6 @@ Past events:
 
 |        Date          |    Time     | Streamers                                             | Show                                                    |
 | :------------------: | :---------: | :---------------------------------------------------- | :------------------------------------------------------ |
-|   Friday, July 22    | 6:00 PM UTC | [Guy Royse][guy]                                      | Web, Whimsy, and Redis Stack: Random Projects with Guy  |
```

Comment:
was this deletion intentional?

Reply:
Yes. My Internet crapped out about 15 minutes into the stream.

Reply:
ahah! gotcha